packages/transaction-log/src/components/common/app.main.component.tsx

```diff
@@ -315,20 +353,140 @@ const App: React.FC = () => {
       setLimit(prevLimit => prevLimit + 20)
 
       await fetchTransactionLogs()
-    } catch (error) {
+    } catch (error: any) {
+      showAlert(
+        'Error fetching more transactions. ' + error?.response?.data,
+        'Error',
+        'error'
+      )
     } finally {
       setLoading(false)
     }
   }
 
+  const handleReRunMatches = async () => {
+    showBackdrop(<Loader />, true)
+
+
+    type BulkRunFilterCountParams = Parameters<typeof getBulkRunFilterCount>;
+
+    const params = {
+      ...getFilters() as any,
+      filterLimit: 1000,
```

Comment:
What happens in a situation we have more than 1000 matches? Has it been handled in the prompt?

Reply:
The legacy app uses 1000 as default.